package.json

```diff
@@ -169,6 +171,7 @@
     "magic-string": "0.26.1",
     "mini-css-extract-plugin": "2.6.0",
     "minimatch": "5.0.1",
+    "minimist": "^1.2.6",
```

Comment:
Can we change the usages of `minimist` with `yargs-parser`? It should be pretty trivial.

Reply:
I'll take a quick look but I'd rather stick with the minimal changes and just get this done asap

Reply:
Well that was even easier then I thought it would be 👍